content/en/cosign/overview.md

```diff
@@ -15,98 +15,96 @@ features:
 
 ![Cosign Overview](/sigstore_cosign-horizontal-color.svg)
 
-Cosign supports container signing, verification, and storage in an OCI registry.
-Cosign aims to make signatures invisible infrastructure.
+## Getting Started (Quick Start)
 
-![Cosign demo gif](/cosign.gif)
+Cosign is part of the Sigstore project. Join us on our [Slack channel](https://sigstore.slack.com/). (Need an [invite](https://links.sigstore.dev/slack-invite)?)
 
-Cosign supports:
+### Installation
 
-<list :items="features" type="info"></list>
+To use Sigstore, you need to install Cosign. Instructions to install Cosign can be found on the [Cosign Installation page](https://docs.sigstore.dev/cosign/installation/). 
 
-Cosign is part of the Sigstore project. Join us on our [Slack channel](https://sigstore.slack.com/) (need an [invite](https://links.sigstore.dev/slack-invite)?)
+This will allow you to sign both blobs and containers.   If you need a sample container to test signing with, you can create one by following [these instructions](https://cloud.google.com/artifact-registry/docs/docker/store-docker-container-images).
```

Comment:
Can we direct towards pulling down an image to sign vs. directing off-site, or else create a page with more direct instructions?

Reply:
I'll just remove the pointer for a sample container.